src/main/java/org/resitrack/entity/Person.java

```diff
@@ -0,0 +1,12 @@
+package org.resitrack.entity;
+
+import org.resitrack.enums.Gender;
+
+import java.time.LocalDate;
+
+public class Person {
+    private String name;
+    private Gender sex;
+    private LocalDate dob;
+    private long idNumber;
```

Comment:
```suggestion
    private String id;
```
the id should a String, we will discuss about the id generation for each person with some random string

Reply:
idNumber is an attribute that represents a person's identity card number
I will add id attribute with data type as String